### DISCUSSION TITLE
Stable release in the near future ?

### DISCUSSION THREAD

**Discussion by somarand** [facebook/lexical#3047] [General]
Hello. This is a great project. Is there any timeline for a stable release ? Any estimates would be cool to have. Thanks !!

**Comment by acywatson** [comment]
Hey! We consider Lexical to be pretty stable - it's in production use on surfaces that are hit by hundreds of millions of users daily. We don't have an exact ETA for a literal semver 1.0, but it should be sometime in early 2023 at the latest

**Reply by WinmezzZ** [reply]
Any plan?

**Reply by AdriatikDushica** [reply]
Hi @acywatson, do you have any news?

Thanks

**Comment by YohanSciubukgian** [comment]
Hello,
The roadmap 2023 describe a potential release to semver 1.0.
As we are now in 2024, do you have any Roadmap update to share and/or maybe potential timeline regarding the 1.0 stable release?
https://github.com/facebook/lexical/wiki/Lexical-Roadmap
